README.md

```diff
@@ -1,13 +1,13 @@
-# Nyaaya V2 Prototype
+# Nyaaya CMS
 
-simple node app to test out features and models for Nyaaya V2
+[Keystone.js](http://keystonejs.com)based CMS for handling the backend of nyaaya-web. 
 
-## Requirements
+## Running locally
 
-    - Node v6.9.4
-    - MongoDB
+	- node version : 8.9.1 LTS; Keystone Version : 4.0beta
```

Comment:
Do we need to link to installation articles? We would need to keep adding them in case we have other dependencies, and the same readme standard should be followed across the repos. Other repos may have more dependencies. Also, these are widely used technologies. 
Added the other changes.